VeganVirus/VeganVirus/ClipBoardAction.h

```diff
@@ -8,15 +8,14 @@
 class ClipBoardAction : public Action
 {
 public:
-	ClipBoardAction(HWND, double);
+	ClipBoardAction(double req);
+
+	virtual void start();
+	virtual void update(double dt);
+
 	void clipBoardWrite(std::string content);
 	std::string ClipBoardRead();
```

Comment:
First, shouldn't this methods be private?
Second, you should write documentation for this methods (and for the constructor)